Tools/ML/MlResponse.h

```diff
@@ -193,13 +222,16 @@ class MlResponse
   }
 
  protected:
-  std::vector<o2::ml::OnnxModel> mModels;         // OnnxModel objects, one for each bin
-  uint8_t mNModels = 1;                           // number of bins
-  uint8_t mNClasses = 3;                          // number of model classes
-  std::vector<double> mBinsLimits = {};           // bin limits of the variable (e.g. pT) used to select which model to use
-  std::vector<std::string> mPaths = {""};         // paths to the models, one for each bin
-  std::vector<int> mCutDir = {};                  // direction of the cuts on the model scores (no cut is also supported)
-  o2::framework::LabeledArray<double> mCuts = {}; // array of cut values to apply on the model scores
+  std::vector<o2::ml::OnnxModel> mModels;              // OnnxModel objects, one for each bin
+  uint8_t mNModels = 1;                                // number of bins
+  uint8_t mNClasses = 3;                               // number of model classes
+  std::vector<double> mBinsLimits = {};                // bin limits of the variable (e.g. pT) used to select which model to use
+  std::vector<std::string> mPaths = {""};              // paths to the models, one for each bin
+  std::vector<int> mCutDir = {};                       // direction of the cuts on the model scores (no cut is also supported)
+  o2::framework::LabeledArray<double> mCuts = {};      // array of cut values to apply on the model scores
+  std::map<std::string, U> mAvailableInputFeatures;    // map of available input features
```

Comment:
Indeed, I just did it to avoid using `static_cast<uint8_t>` when filling `mAvailableFeatures` or comparing an `enum` entry and a `mCachedIndices` entry (in `getInputFeatures`). But I can add those and explicitely put the type `uint8_t`.

Reply:
Actually, doing this (and also documenting the `template` of the classes as adivsed by Vit), I realized that it might be better to change from `typename U = uint8_t` to `class EnumInputFeatures = uint8_t` and go from `enum` to `enum class` for the available input features.

This way, it makes it a bit more clear that this template argument expects a whole `enum` instance and not just a type (while keeping `typename` could me misleading). I tested it and it works, so just tell me @fcatalan92 , @vkucera , which version you prefer (here in the example of D+):
- `typename EnumInputFeatures = InputFeaturesDplusToPiKPi` with `enum InputFeaturesDplusToPiKPi`
- or `class EnumInputFeatures = InputFeaturesDplusToPiKPi` with `enum class InputFeaturesDplusToPiKPi`
?

PS: knowing that the default value of this `template` can still be set to `uint8_t`.

Reply:
After discussion with Fabio, I implemented the second option with `enum class`.